### PR TITLE
ISPN-6758: JDBC DB Major/Minor Versions not exposed by the server.

### PR DESCRIPTION
**Pull request by ryanemerson**
https://issues.jboss.org/browse/ISPN-6758

### CODE REVIEW

**Comment by tristantarrant** [comment]
You should add them to the parser, as well as to the subsystem test case

**Comment by tristantarrant** [comment]
And don't forget CacheConfigurationAdd

**Comment by ryanemerson** [comment]
@tristantarrant Done (I think).

File: server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheConfigurationAdd.java [hunk 1]
```diff
@@ -865,6 +865,16 @@ public void inject(OutboundSocketBinding value) {
         }
     }
 
+   private void buildDbVersions(AbstractJdbcStoreConfigurationBuilder builder, OperationContext context, ModelNode store) throws OperationFailedException {
+      ModelNode dbMajorVersion = BaseJDBCStoreConfigurationResource.DB_MAJOR_VERSION.resolveModelAttribute(context, store);
```
Comment (slaskawi) [inline server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheConfigurationAdd.java]: What is the advantage of using `resolveModelAttribute` over getting the value directly (`store.get`.. kind of operations)?

Comment (tristantarrant) [inline server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheConfigurationAdd.java]: It resolves expressions

**Comment by tristantarrant** [comment]
Pushed to master, thanks

**Pull request closed without merging**